releasenotes/notes/fix-ossa-2024-004-f732e58c12e26785.yaml

```diff
@@ -0,0 +1,6 @@
+---
+critical:
```

Comment:
security?

Reply:
https://docs.openstack.org/reno/latest/user/usage.html#editing-a-release-note - list of default categories

Reply:
I was following the same layout as done here.

https://github.com/stackhpc/stackhpc-kayobe-config/pull/1186/files